<?xml version="1.0"?>
<launch>

	<group ns="frobo">
		<node pkg="microcontroller_connector" name="connector" type="connector.py" output="screen"> </node>  
		<!--
		<node pkg="frobo" name="control" type="control" output="screen"> 
			<param name="object_topic" value ="object_topic" />
			<param name="object_row_topic" value ="object_row_topic" />
			--><!--<param name="fpga_data" value ="encoder_l" />
			<param name="pwm" value ="pwm" />
			<param name="state" value ="state" />
			<param name="turn90" value = "1050" />
			<param name="exitRowTicks" value = "1800" />
			<param name="returnTicks" value= "6000" />
			<param name="moveSpeed" value = "10" />
			<param name="Preg" value = "100"/>
			<param name="regSpeed" value = "20"/>--> <!--
		</node>
		 <node pkg="frobo" type="pilot_node" name="pilot_node" output="screen"> 
			<param name="laser_scan_topic" value="laser_msgs_1" />
			<param name="show_image" value="0" />
			<param name="object_topic" value="object_topic" />
			<param name="object_row_topic" value="object_row_topic" /> -->
			<!--<param name="wheel_speeds_topic" value="/Hilde/wheel_speed_topic" /> --> <!--
			<param name="robot_clearence_width" value="0.15" />
			<param name="robot_stop_zone" value="0.02" />
			<param name="robot_turn_zone" value="0.8" /> 
			<param name="robot_turn_zone_extra_width" value="-0.05" />
			
			<param name="row_box_start_value" value="0.03" />
			<param name="row_box_width" value="0.30" />
			<param name="row_box_height" value="1.1" />
			<param name="row_box_count" value="10" />
		</node>-->
		
		<node pkg="frobo" name="lms111_front" type="lms111_node" output="screen">
			<param name="host" value="10.0.42.75"/>
			<param name="topic_id" value="laser_msgs_1"/>
			<param name="frame_id" value="base_laser"/>-
		</node>
		
	</group>

	
</launch>
